File: week04-2.cpp
//week04-2.c 琌さぱ Leetcode 珼驹 Easy 琌()
//Leetcode 2529. Maxium Count of Postive Integer and Negative Integer
int maximumCount(int* nums, int numsSize) {
    int pos = 0, neg = 0;// 癹伴玡,非称,常0

    for(int i=0; i<numsSize; i++) {
        if( nums[i] > 0 ) pos++; // タ计
        if( nums[i] < 0 ) neg++; // 璽计
    }//癹伴い丁穝эウ
    //癹伴рウㄓノ
    if(pos>neg) return pos; //タ计ゑ耕,タ计
    else return neg; // ぃ礛,碞璽计
    }

File: week04-2.py
# week04-2.py 是今天的 Leetcode 挑戰題  easy 題(二合一)
# Leetcode 2529. Maximum count of Positive Integer
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0 # 迴圈前面 準備好,都放0
        for i in range(len(nums)): # 迴圈哩，更新它
            if nums[i] > 0:pos += 1
            if nums[i] < 0:neg += 1
        # 迴圈後面，把答案拿來用
        return max(pos, neg)
